dags/atd_service_bot_issues_to_socrata.py

```diff
@@ -62,16 +63,26 @@
     dagrun_timeout=timedelta(minutes=60),
     tags=["repo:atd-service-bot", "socrata", "github"],
     catchup=False,
+    render_template_as_native_obj=True,
 ) as dag:
+    @task(
+        task_id="get_env_vars",
+        execution_timeout=timedelta(seconds=30),
+    )
+    def get_env_vars():
+        from utils.onepassword import load_dict
+        env_vars = load_dict(REQUIRED_SECRETS)
+        return env_vars
+
     t1 = DockerOperator(
         task_id="dts_github_to_socrata",
         image=docker_image,
         api_version="auto",
         auto_remove=True,
         command="./atd-service-bot/issues_to_socrata.py",
-        environment=env_vars,
+        environment="{{ task_instance.xcom_pull(task_ids='get_env_vars') }}",
```

Comment:
by default the `task_instance.xcom_pull()` method is going to pull from the `return_value` key. so this template variables pipes our secrets dictionary right into the Docker environment.